### PR TITLE
Thread dashboard pull request manually merged and fixed

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**

### CODE REVIEW

**Comment by syoung-smallwisdom** [comment]
Note: I have fixed the merge conflicts and run on device.  Also, did a quick pass at a code review. Looks good.

File: mPowerSDK/Dashboard/APHDashboardViewController.m [hunk 1]
```diff
@@ -34,9 +34,16 @@
...
+#import "APHDashboardGraphTableViewCell.h"
 #import "APHDataKeys.h"
 #import "APHLocalization.h"
+#import "APHMedicationTrackerTask.h"
+#import "APHMedicationTrackerViewController.h"
 #import "APHSpatialSpanMemoryGameViewController.h"
+#import "APHScatterGraphView.h"
+#import "APHTableViewDashboardGraphItem.h"
+#import "APHScoring.h"
+//#import "APHTremorTaskViewController.h"
```
Comment (Erin-Mounts) [inline mPowerSDK/Dashboard/APHDashboardViewController.m]: Does this need to be un-commented once the tremor task is merged back in?

**Comment by evliu** [comment]
Thanks a lot! I'll check w/ Jake about the commenting.

File: mPowerSDK/Dashboard/APHDashboardViewController.m [hunk 2]
```diff
@@ -497,13 +516,42 @@ - (void)prepareData
                     [rowItems addObject:row];
                 }
                     break;
+                    
+                /*
```
Comment (Erin-Mounts) [inline mPowerSDK/Dashboard/APHDashboardViewController.m]: see above comment about tremor task and uncommenting; also, mood section shouldn't be commented-out.

**Comment by syoung-smallwisdom** [comment]
@Erin-Mounts I have fixed the merge issues introduced with the tremor module changes and commented out the tremor module from the dashboard. Should be good to go. Passed a quick smoke test on device.

**Pull request merged**